ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa_hcp-4.15-nightly-x86.yaml

```diff
@@ -73,6 +73,30 @@ tests:
     - chain: openshift-qe-node-density-heavy
     workflow: rosa-aws-sts-hcp
   timeout: 5h0m0s
+- always_run: false
+  as: node-density-24nodes
```

Comment:
Why are running node-density lite outside of the "control plane" cdv2 + lite combo?

Reply:
Oh I also noticed it is both cron and not-cron here, and not-cron in the 4.16 config. So maybe this one is meant to be not-cron?

Reply:
It was added for debugging @afcollins - I turned off cron for all node-density lite now